### DISCUSSION TITLE
How to add my own `ClientType`?

### DISCUSSION THREAD

**Discussion by ImJeremyHe** [informalsystems/ibc-rs#185] [Q&A] [answered]
I am working on implementing my own `ClientState`. But the `client_type` method returns `ClientType` only including tendermint.

**Comment by plafer** [comment, marked as answer]
We completely missed that, thank you for reporting! Created this [issue](https://github.com/cosmos/ibc-rs/issues/188) as a result, and will start working on it now.

**Reply by plafer** [reply]
Fixed in `v0.21.0`.
